tmt/steps/report/reportportal.py

```diff
@@ -1,152 +1,221 @@
 import dataclasses
-import io
 import os
-import types
-import zipfile
 from typing import Optional
 
-import tmt.steps.report
-import tmt.utils
-
-from . import junit
-
-junit_xml: Optional[types.ModuleType] = None
-
+import requests
 
-def import_junit_xml() -> None:
-    """
-    Import junit_xml module only when needed
-
-    Until we have a separate package for each plugin.
-    """
-    global junit_xml
-    try:
-        import junit_xml
-    except ImportError:
-        raise tmt.utils.ReportError(
-            "Missing 'junit-xml', fixable by 'pip install tmt[report-junit]'.")
+import tmt.steps.report
+from tmt.result import ResultOutcome
+from tmt.utils import field, yaml_to_dict
 
 
 @dataclasses.dataclass
 class ReportReportPortalData(tmt.steps.report.ReportStepData):
-    url: Optional[str] = tmt.utils.field(
+    url: Optional[str] = field(
         option="--url",
         metavar="URL",
         default=os.environ.get("TMT_REPORT_REPORTPORTAL_URL"),
         help="The URL of the ReportPortal instance where the data should be sent to.")
-    token: Optional[str] = tmt.utils.field(
+    token: Optional[str] = field(
         option="--token",
         metavar="TOKEN",
         default=os.environ.get("TMT_REPORT_REPORTPORTAL_TOKEN"),
-        help="The token to use for upload to the ReportPortal instance.")
-    project: Optional[str] = tmt.utils.field(
+        help="The token to use for upload to the ReportPortal instance (from RP user profile).")
```

Comment:
```suggestion
        help="The token to use for upload to the ReportPortal instance (from the user profile).")
```

Let's not use the `RP` abbreviation.